tests/steps/test-discover-repos.sh

```diff
@@ -30,12 +30,9 @@ tex=${TARGET}/foo.tex
 
 {
     rm -f "${csv}"
-    msg=$("${LOCAL}/steps/discover-repos.rb" --dry --pause=0 --total=3 --min-stars=100 --max-stars=1000 "--csv=${csv}"  "--tex=${tex}")
+    msg=$("${LOCAL}/steps/discover-repos.rb" --dry --pause=0 --total=102 --min-stars=100 --max-stars=1000 "--csv=${csv}"  "--tex=${tex}")
     echo "${msg}"
-    echo "${msg}" | grep "Found 1 repositories in page #0"
```

Comment:
@RuslanGaliullin in general, it's not a good practice to delete test assertions. You are weakening the test, which makes your pull request look suspicious: maybe you break some existing functionality? Would be better if you _add_ a new test scenario below the existing one.

Also, I think that the download of 102 repositories will make the test rather slow. Maybe it's better to decrease the size of the page to, say, 10: `--total=12 --page-size=10`